install-and-hook.ps1

```diff
@@ -0,0 +1,4 @@
+$ts=Get-Date -Format o | ForEach-Object {$_ -replace ":", "."}
```

Comment:
Get-Date should have a cleaner way of returning the date with periods.
If not you can also use this statement as well to replace the colons with periods.
$ts = (Get-Date -Format o).replace(":",".")

Reply:
Done. changed to `(Get-Date -Format o).replace(":",".")`